android/app/src/main/java/labs/pooh/eaterslab/ui/activity/map/MapVewModel.kt

```diff
@@ -7,12 +7,12 @@ import androidx.lifecycle.viewModelScope
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.launch
 import kotlinx.coroutines.withContext
-import labs.pooh.client.apis.CafeteriaApi
+import labs.pooh.client.apis.CafeteriasApi
```

Comment:
Looks like some extra code got in.

Reply:
api changed on server side so to get apk compiled these canges had to be done

Reply:
Then it should be a separate PR, it has nothing to do with APK builds.